README.md

```diff
@@ -183,34 +183,47 @@ has(key)
 
 ```js
 // Returns an array of all the values stored in the Map.
+// This will order values if before() or after() are used.
 // returns: Array
 values()
 ```
 
 ```js
 // Returns an object of all the entries in the backing Map
 // where the key is the object property, and the value
-// corresponding to the key. Will return `undefined` if the backing 
-// Map is empty.
+// corresponding to the key.
+// Will return `undefined` if the backing Map is empty.
+// This will order keys if before() or after() are used.
 // returns: Object, undefined if empty
 entries()
 ````
 
 ```js
 // Provide an object which maps its properties and values 
-// into the backing Map as keys and values.
+// into the backing Map as keys and values. Can specify
+// an array of keys to omit from the merge.
```

Comment:
When I read this I was imagining the keys being omitted might be those on the original object (eg so a new array could be passed in for a particular property and not just merged in with the existing array). However from reading the source below the opposite appears to be the case? (I'm struggling at quick glance to tell though due to the larger diff and not having spent long looking at it) Either way I think it would be good to clarify in this comment :-)

Reply:
Yeah, I'll try to phrase this better. The `omit` property is supposed to be an array of keys that should be excluded from merging into the config, e.g. the following would do nothing.

```js
config.merge({ devtool: 'source-map' }, ['devtool'])
```